src/components/SearchForm.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <form
+    class="search-form"
+    @submit.prevent="onSubmit"
+  >
+    <img
+      class="search-icon"
+      src="../assets/search.svg"
+      alt="Search"
+      width="20"
+    >
+    <input
+      v-model.trim="searchValue"
+      class="search-input"
+      type="text"
+      placeholder="Search"
+    >
+    <label class="search-adult">  
+      <span>Include adult</span>
+      <input
+        v-model="adultIncluded"
+        type="checkbox"
+      >
+    </label>
+  </form>
+</template>
+
+<script>
+export default {
+  name: 'SearchComponent',
+  data() {
+    return {
+      searchValue: '',
+      prevSearchValue: '',
+      adultIncluded: false,
+    };
+  },
+  methods: {
+    onSubmit() {
+      if (this.searchValue !== '' && this.searchValue !== this.prevSearchValue) {
```

Comment:
`prevSearchValue` doesn't work properly. When you search for something, and then click on `adultIncluded` checkbox, this condition doesn't alow you to search again, because `searchValue` has not changed. 

Also `this.searchValue !== ''` should be  replaced to just `this.searchValue`, it will work the same, but the code will be cleaner